packages/medusa/src/services/order-edit.ts

```diff
@@ -0,0 +1,51 @@
+import { EntityManager } from "typeorm"
+import { MedusaError } from "medusa-core-utils"
+import { OrderEditRepository } from "../repositories/order-edit"
+import { OrderEditStatus } from "../models/order-edit"
+import { TransactionBaseService } from "../interfaces"
+
+type InjectedDependencies = {
+  manager: EntityManager
+  orderEditRepository: typeof OrderEditRepository
+}
+
+export class OrderEditService extends TransactionBaseService {
+  protected transactionManager_: EntityManager | undefined
+  protected readonly manager_: EntityManager
+
+  protected readonly orderEditRepository_: typeof OrderEditRepository
+
+  constructor({ manager, orderEditRepository }: InjectedDependencies) {
+    super({ manager, orderEditRepository })
+
+    this.manager_ = manager
+    this.orderEditRepository_ = orderEditRepository
+  }
+
+  async delete(orderEditId: string): Promise<void> {
+    return this.atomicPhase_(async (manager) => {
+      const orderEditRepo = manager.getCustomRepository(
+        this.orderEditRepository_
+      )
+
+      const edit = await orderEditRepo.findOne({ where: { id: orderEditId } })
```

Comment:
suggestion: Here we need to retrieve the active order edit. multiple order edit can be created for one order. I would suggest to implement the following method https://github.com/medusajs/medusa/pull/2187/files#diff-a26e080c56da65f0067681a51f853510451acbbe4700216d78a333826a71357dR63 wdyt? or something similar I let you decide. But this will also be needed for the creation as you can see